deploy/cr.yaml

```diff
@@ -122,6 +136,36 @@ spec:
 #          memory: 1G
 #          cpu: 700m
 
+#      env:
+#      - name: HA_CONNECTION_TIMEOUT
+#        value: "1000"
+#
+#      envFrom:
+#      - secretRef:
+#          name: haproxy-env-secret
+
+      readinessProbe:
+        timeoutSeconds: 3
+        periodSeconds: 5
+        failureThreshold: 3
+        successThreshold: 1
+
+      livenessProbe:
+        timeoutSeconds: 3
+        periodSeconds: 5
+        failureThreshold: 3
+        successThreshold: 1
+
+#      configuration: |
```

Comment:
Please check PXC cr and provide the link conf to have an example of custom config. The support team asked about it.

Reply:
done